internal/playwright/config.go

```diff
@@ -141,3 +147,40 @@ func SplitSuites(p Project) (Project, Project) {
 
 	return dockerProject, sauceProject
 }
+
+func checkSupportedBrowsers(p *Project) error {
+	supportedBrwsList := []string{"chromium", "firefox", "webkit"}
+	errMsg := "browserName: %s is not supported. List of supported browsers: %s"
+
+	if p.Playwright.Params.BrowserName != "" {
+		found := false
+		for _, browser := range supportedBrwsList {
+			if p.Playwright.Params.BrowserName == browser {
+				found = true
+				break
+			}
+		}
+
```

Comment:
@valentyn88 we may want to move this to its own method, like `isBrowserSupported` so you can use it again in the other check.

Reply:
In this case, I need to pass an `interface{}` instead of `Project` and use types assertion. Are you ok with this approach?